ui/pages/confirm-approve/confirm-approve.js

```diff
@@ -163,101 +181,106 @@ export default function ConfirmApprove() {
-            tokenImage={tokenImage}
-            tokenBalance={tokenBalance}
-            showCustomizeGasModal={approveTransaction}
-            showEditApprovalPermissionModal={({
-              /* eslint-disable no-shadow */
-              customTokenAmount,
-              decimals,
-              origin,
-              setCustomAmount,
-              tokenAmount,
-              tokenBalance,
-              tokenSymbol,
-              /* eslint-enable no-shadow */
-            }) =>
-              dispatch(
-                showModal({
-                  name: 'EDIT_APPROVAL_PERMISSION',
-                  customTokenAmount,
-                  decimals,
-                  origin,
-                  setCustomAmount,
-                  tokenAmount,
-                  tokenBalance,
-                  tokenSymbol,
-                }),
-              )
-            }
-            data={customData || data}
-            toAddress={toAddress}
-            currentCurrency={currentCurrency}
-            nativeCurrency={nativeCurrency}
-            ethTransactionTotal={ethTransactionTotal}
-            fiatTransactionTotal={fiatTransactionTotal}
-            hexTransactionTotal={hexTransactionTotal}
-            useNonceField={useNonceField}
-            nextNonce={nextNonce}
-            customNonceValue={customNonceValue}
-            updateCustomNonce={(value) => {
-              dispatch(updateCustomNonce(value));
-            }}
-            getNextNonce={() => dispatch(getNextNonce())}
-            showCustomizeNonceModal={({
-              /* eslint-disable no-shadow */
-              useNonceField,
-              nextNonce,
-              customNonceValue,
-              updateCustomNonce,
-              getNextNonce,
-              /* eslint-disable no-shadow */
-            }) =>
-              dispatch(
-                showModal({
-                  name: 'CUSTOMIZE_NONCE',
-                  useNonceField,
-                  nextNonce,
-                  customNonceValue,
-                  updateCustomNonce,
-                  getNextNonce,
-                }),
-              )
-            }
-            warning={submitWarning}
-            txData={transaction}
-            fromAddressIsLedger={fromAddressIsLedger}
-            chainId={chainId}
-            rpcPrefs={rpcPrefs}
-            isContract={isContract}
-            isMultiLayerFeeNetwork={isMultiLayerFeeNetwork}
-          />
-          {showCustomizeGasPopover && (
-            <EditGasPopover
-              onClose={closeCustomizeGasPopover}
-              mode={EDIT_GAS_MODES.MODIFY_IN_PLACE}
-              transaction={transaction}
+    <GasFeeContextProvider transaction={transaction}>
+      <ConfirmTransactionBase
+        toAddress={toAddress}
+        identiconAddress={tokenAddress}
+        showAccountInHeader
+        title={tokensText}
+        contentComponent={
+          <TransactionModalContextProvider captureEventEnabled={false}>
```

Comment:
`captureEventEnabled={false}` — was this moved from somewhere else or is this a new thing? What does this do?

Reply:
This is new route and different flow - approve token. Please check video in description.